tests/test_metadata_eq_.py

```diff
@@ -182,6 +183,23 @@ def test_snapshot_eq_(self) -> None:
         setattr(signed_copy, "meta", None)
         self.assertNotEqual(md.signed, signed_copy)
 
+    def test_hash_delegations_eq(self) -> None:
+        succinct_hash_info_dict = {
+            "delegation_hash_prefix_len": 8,
+            "bin_name_prefix": "delegated",
+        }
+        hash_deleg = SuccinctHashDelegations.from_dict(succinct_hash_info_dict)
```

Comment:
not terribly important but I think it's weird to use `from_dict()` outside of serialization testing: from_dict() is a serialization implementation detail, and using the constructor should be less lines of code as well.

This style is used in multiple tests